docs/pages/quickstart_guide.mdx

```diff
@@ -0,0 +1,78 @@
+
+
+**Federations:**
+Objects in Pelican belong to *federations*, which are aggregations of data that are exposed to other individuals in the federation. Each Pelican federation constitutes its own global namespace of objects and each object within a federation has its own path, much like files on a computer. Fetching any object from a federation requires at minimum two pieces of information: a URL indicating the object's federation and the path to the object within that federation (there is the potential that some objects require access tokens as well, but more on that later). For example, the OSDF’s federation URL is https://osg-htc.org and an example object from the federation can be found at the following path:
+```console
+/ospool/uc-shared/public/OSG-Staff/validation/test.txt
+```
+**Namespace Prefixes:**
+Each origin supports one or more *namespace prefixes*, which are analogous to the folders or directories from your computer that you use to organize files. Below is an example object:
+```bash
+/demo/testfile.txt
+```
+This is a simple example where the namespace prefix would be `/demo` and the object is named `/testfile.txt`.
+
+However, things can get a little tricky when you add a longer namespace prefix and/or a longer filepath within that namespace where the object lives. For example:
+```bash
+/ospool/uc-shared/public/OSG-Staff/validation/test.txt
+```
+In this example, the namespace prefix is `/ospool/uc-shared/public/`, and the actual object is named `/OSG-Staff/validation/test.txt`. While the namespace prefix always comes before the object name, it can get a little confusing once paths and names get longer.
+
+
+## Your First Pelican Client Command
+Accessing objects from a Pelican federation requires the use of the Pelican client commands. Here we will be using the `pelican object get` command to get an object from a federation. This command is formatted like so:
+```console
+pelican object get pelican://<federation-url>/<namespace-prefix>/<path/to/federation/object> </local/path/to/destination>
+```
+We can break down the components of this command before you try it yourself:
+- `pelican object get`: This is the client command we are utilizing. `object` because we want to interact with Pelican objects and `get` because we want to GET the file from our federation.
+- `pelican://`: This is the URL scheme for accessing this object. Pelican accepts different URL schemes which are detailed in the [client usage](./client_usage.mdx#the-different-pelican-url-schemes) page.
+- `<federation-url>`: The URL indicating the object's location.
+- `<namespace-prefix>`: The namespace at which this object is located.
```

Comment:
Not a request for a change, per say, but I think we should think carefully about `namespace prefix`. WRT to origin configuration, we've changed the name to `FederationPrefix`, and I wonder if we should start consolidating the language before we wind up with a messy plate of name spaghetti. For now, let's keep it as is, but let's also bring this up soon.

Reply:
Sounds good, good point!